disease-classification-training/test_train.py

```diff
@@ -0,0 +1,72 @@
+import os, sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))) 
+import numpy as np
+import unittest
+import yaml
+from train import train_model
+from yaml.loader import SafeLoader
+
+YAML_ARG_TO_TEST = 'test_arguments'
+RANDOM_SEED = 42
+
+np.random.seed(RANDOM_SEED)
+
+class TestDiseaseClassification(unittest.TestCase):
+    def setUp(self) -> None:
+        cfg_path = os.path.dirname(os.path.abspath(__file__))
+        cfg_file = cfg_path + "/" + "test_config.yaml"
+        self.test_cfg = {}
+        with open(cfg_file) as c_info_file:
+            self.test_cfg = yaml.load(c_info_file, Loader=SafeLoader)
+        self.test_cfg = self.test_cfg[YAML_ARG_TO_TEST]
+        self.classification_df, self.confusion_df = train_model(
+            lr=self.test_cfg["lr"],
+            epochs=self.test_cfg["epochs"],
+            batch=self.test_cfg["batch_size"],
+            model_name=self.test_cfg["model_name"],
+            output_dir=self.test_cfg["output_dir"]
```

Comment:
Same key error for output_dir as well

Reply:
I was using an old config file, fixed the config file to address the issue. Thank you!